terraform/gce/gce.tf

```diff
@@ -31,18 +34,12 @@ resource "google_compute_firewall" "mi-firewall-external" {
     protocol = "icmp"
   }
 
-  # RDP
   allow {
     protocol = "tcp"
     ports = [
-      "22",   # SSH
```

Comment:
did we decide to close off mesos, marathon, and consul ports? if so, should we make AWS consistent?